app/views/layouts/_topbar.html.erb

```diff
@@ -9,7 +9,7 @@
       <ul class="top-nav-list">
         <% if current_user %>
           <li><%= "Logged in as #{current_user.username}" %></li>
-          <li><%= link_to "Register Organization", new_organization_path %></li>
+          <li><%= link_to "Register Organization", new_organization_path  unless org_admin?%></li>
           <li id="cartCounter"><span class="glyphicon glyphicon-shopping-cart" aria-hidden="true"></span><%= link_to "Cart: $#{@cart.total}", cart_path %> <span class="glyphicon glyphicon-log-out" aria-hidden="true"></span><%= link_to "Logout", logout_path, method: :delete %></li>
```

Comment:
@Caleb9193 I think you need to set up a controller action or take away the permission for the org admin as well